test/e2e/README.md

```diff
@@ -1,83 +1,52 @@
 # ORAS End-to-End Testing Dev Guide
 **KNOWN LIMITATION**: E2E tests are designed to run in the CI and currently only support running on linux platform.
-## Setting up
-Minimal setup: Run the script in **step 3**
+## Prerequisites
+Install [git](https://git-scm.com/downloads), [docker](https://docs.docker.com/desktop/install/linux-install/), [go](https://golang.google.cn/dl/).
```

Comment:
Those links are strange.

The 1st link is for Windows but the 2nd link is for Linux. Meanwhile, the 3rd link is China specific.

Reply:
Good catch! Will resolve.